ReadsUtils.spec

```diff
@@ -69,6 +69,13 @@ module ReadsUtils {
         - OR -
         fwd_file - a local path to the reads data file: either single end
             reads, forward/left reads, or interleaved reads.
+        - OR - 
+        fwd_file_url - a download link that contains reads data file:
+            either single end reads, forward/left reads, or interleaved reads.
+        download_type - download type ['Direct Download', 'FTP', 'DropBox', 'Google Drive']
+        - OR - 
+        fwd_staging_file_name - reads data file name in staging area:
+            either single end reads, forward/left reads, or interleaved reads.
```

Comment:
See lines 56-60 - they assume only 2 input types are now available instead of 4 (plus 4 download types, so 7 really)